From 9f8f5fbe077de1633901ca9bd02dada6dfe71332 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Fri, 28 Oct 2016 03:23:16 +0100
Subject: [PATCH] applets/icon-tasklist: Remove redundant "Launch" label, Pedro
 was right

Think about it, it's a launcher. What else was it gonna do, sing?

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/panel/applets/icon-tasklist/Buttons.vala | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/panel/applets/icon-tasklist/Buttons.vala b/panel/applets/icon-tasklist/Buttons.vala
index b3b243e..8004155 100644
--- a/panel/applets/icon-tasklist/Buttons.vala
+++ b/panel/applets/icon-tasklist/Buttons.vala
@@ -448,7 +448,7 @@ public class PinnedIconButton : IconButton
         this.settings = settings;
 
         this.context = context;
-        set_tooltip_text("Launch %s".printf(info.get_display_name()));
+        set_tooltip_text(info.get_display_name());
         image.set_from_gicon(info.get_icon(), Gtk.IconSize.INVALID);
 
         alt_menu = new Gtk.Menu();
@@ -519,7 +519,7 @@ public class PinnedIconButton : IconButton
     public void reset()
     {
         image.set_from_gicon(app_info.get_icon(), Gtk.IconSize.INVALID);
-        set_tooltip_text("Launch %s".printf(app_info.get_display_name()));
+        set_tooltip_text(app_info.get_display_name());
         get_style_context().remove_class("running");
         set_active(false);
         // Actions menu
